frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -44,6 +49,7 @@ const ModelDetailsPage: React.FC = conditionalArea(
   const decodedParams = decodeParams(params);
   const { modelRegistryServices, modelRegistryServicesLoaded, modelRegistryServicesLoadError } =
     React.useContext(ModelRegistrySelectorContext);
+  const tuningAvailable = !useIsAreaAvailable(SupportedArea.FINE_TUNING).status;
```

Comment:
This value is the reverse of what we want - the `!` here makes this true of tuning is not available. You can see and change the values of the feature flags by putting `?devFeatureFlags` at the end of the URL and clicking the override link that appears, it was disabled on ui-green. I just enabled it for us.

Reply:
think that may have actually been a sloppy lint fix of some sort, 😕 i was using the flag.